h/static/scripts/annotation-sync.coffee

```diff
@@ -130,6 +130,15 @@ module.exports = class AnnotationSync
       return unless bodies.length
       @bridge.call('loadAnnotations', bodies)
 
+    'annotationsUnloaded': (annotations) ->
+      self = this
+      annotations.forEach (annotation) ->
+        # In the client, unloading an annotation is handled the same way as
+        # deleting an annotation. Within the app however, we handle the events
+        # differently in some cases
```

Comment:
Good point. By 'client' here I mean "the annotator guest that lives in the page" and by 'app' I mean, "the app running in the sidebar". These are referred to as "guest" and "server" in some places. Consistent vocabulary would be useful though.